### ISSUE TITLE
name 'smi_out' is not defined - fast_stable_diffusion notebook

### ISSUE THREAD

**Issue by MasterDenis** [TheLastBen/fast-stable-diffusion#271]
Info Here:
![image](https://user-images.githubusercontent.com/61897713/198545880-9d928a09-5ce3-4e4f-9a60-5b1f6b4cb60d.png)

**Comment by EntityinArray** [comment]
Happens for me too

**Comment by EntityinArray** [comment]
Commit that causes the issue: https://github.com/TheLastBen/fast-stable-diffusion/commit/6a44c8159d8c795a32f960b1d5bc618a93708b74

**Comment by elBlacksmith** [comment]
> Commit that causes the issue: [6a44c81](https://github.com/TheLastBen/fast-stable-diffusion/commit/6a44c8159d8c795a32f960b1d5bc618a93708b74)

Thanks for pointing this, it´s not hard to change on Colab as temporal fix.

**Comment by TheLastBen** [comment]
will fix it in a bit

**Comment by ejcho623** [comment]
<img width="1107" alt="Screen Shot 2022-10-29 at 9 59 51 AM" src="https://user-images.githubusercontent.com/5448470/198843871-c6c30894-9783-4d88-a218-0b6d81ebe093.png">
Getting an error on this line now

**Comment by TheLastBen** [comment]
did you run the cell before ?

**Comment by ejcho623** [comment]
yeah i did.. although i've tried again by restarting the runtime, and got it to run, thanks.
